keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -491,37 +456,30 @@ Defined using feature gate
 - Plaintext output roughly matches explain output
 - OpenAPIV3 (raw json) output implemented
 
-#### Alpha 2
-
-If we decide to move ahead with the `md` and `html` outputs, an Alpha 2 may
-be required.
-
-- `md` output implemented (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Section for each individual GVK
-  - All types hyperlink to specific section
--  basic `html` output  (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Page for each individual GVK.
-  - All types hyperlink to their specific page
-  - Searchable by name, description, field name.
-
 #### Beta
 
 - kube-openAPI v3 JSON deserialization is optimized to take less than 150ms on
   most machines
+    - No longer a goal for beta - we don't use kube-openapi types in this 
+      implementation. However a benchmark should still be written to measure
+      deserialization into the type used `map[string]any`
```

Comment:
Yeah, also that made sense back then because we had not fixed the json deserialization of openapi type. Agreed with you anyway, completely obsolete.